cmd/crossbuild_tarballs.go

```diff
@@ -53,7 +54,8 @@ func runCrossbuildTarballs() {
 			os.Setenv("GOOS", platform[0])
 			os.Setenv("GOARCH", platform[1])
 		} else {
-			fatalMsg(nil, fmt.Sprintf("Bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform))
+			err := errors.New(fmt.Sprintf("Bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform))
```

Comment:
`fmt.Errorsf("bad .build/%s directory naming, should be <GOOS>-<GOARCH>", platform)`